node/src/node/slave/machine.cpp

```diff
@@ -433,6 +449,17 @@ machine_t::dump() {
     });
 }
 
+machine_t::ewma_type &
```

Comment:
It takes some time to resolve "align pointers" statement and to check code on "pointers alignment policy". :smile:

fixed in 51b63ff65098d8fc4e49b72c180decc61c8b9ff0